.eslintignore

```diff
@@ -3,3 +3,4 @@ build
 my-app*
 packages/react-scripts/template
 packages/react-scripts/fixtures
+packages/react-dev-utils/*.js
```

Comment:
![screen shot 2017-08-23 at 8 51 37 pm 1](https://user-images.githubusercontent.com/9636410/29619527-86bd9aa6-8845-11e7-8f40-46abdd9d97fd.png)
have it here, I think GitHub truncate it?